WeSolve/frontend/src/components/Question.vue

```diff
@@ -160,22 +166,35 @@ export default {
     },
     onSubmit() {
       // Tell the REST API to create a new answer for this question based on the user input, then update some data properties
-      let answerData = {};
+      let formData = new FormData();
       if (!this.newAnswerBody && !this.answerPDF) {
         this.error = "You can't send an empty answer!";
       } else {
         if (this.newAnswerBody) {
-          answerData.body = this.newAnswerBody;
+          formData.append("body", this.newAnswerBody);
         }
         if (this.answerPDF) {
-          answerData.answerPDF = this.answerPDF;
+          formData.append("answerPDF",this.answerPDF);
+        }
+
+        let config = {
+          headers: {
+            'X-CSRFTOKEN': CSRF_TOKEN
+          }
         }
-        let endpoint = `/api/questions/${this.slug}/answer/`;
-        apiService(endpoint, "POST", answerData)
-            .then(data => {
+
+        axios
+          .post(`/api/questions/${this.slug}/answer/`, formData, config)
+          .then(data => {
               this.answers.unshift(data)
-            })
+            });
+
         this.newAnswerBody = null;
+        this.answerPDF = null;
+        this.form = {
+                body: "",
+                answerPDF: null,
+        };
```

Comment:
remove  ';'    ????
The ';' helps the "You've written an answer!" pops immediately after pushing the bottom